nodejs/week3/node.js-week3-Homework/calculator/index.js

```diff
@@ -0,0 +1,65 @@
+const express = require("express");
+const calculator = express();
+const port = 3000;
+calculator.use(express.json());
+calculator.use(express.urlencoded({ extended: true }));
+
+// Calculator using :method (path) parameter:
+calculator.get("/calculator/:method", (request, response) => {
+	const queryParams = Object.values(request.query);
+	const numsMerged = [].concat.apply([], queryParams);
```

Comment:
I believe numsMerged variable is not necessary. 
const queryParams is itself an array, so you can do .map method on it.